### PR TITLE
Release v0.4.4

### PR DESCRIPTION
**Pull request by jacquelynsmale**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [PEP 440](https://www.python.org/dev/peps/pep-0440/) 
 and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.4.4]
+
+### Changed
+* `asf-tools.water_map` now produces water extent rasters with pixels values of `1` where water is present and `0` where water is not present
+* `asf_tools.water_map` now uses the  updated [ASF Global HAND dataset](https://copernicus-hand-30m.s3.amazonaws.com/) derived from the [2021 release of the Copernicus GLO-30 Public DEM](https://spacedata.copernicus.eu/blogs/-/blogs/copernicus-dem-2021-release-now-available)
```
Comment (jhkennedy) [inline CHANGELOG.md:13]:
```suggestion
* `asf_tools.water_map` now uses the  updated [ASF Global HAND dataset](https://glo-30-hand.s3.amazonaws.com/readme.html) derived from the [2021 release of the Copernicus GLO-30 Public DEM](https://spacedata.copernicus.eu/blogs/-/blogs/copernicus-dem-2021-release-now-available)
```

File: asf_tools/water_map.py [hunk 2]
```diff
@@ -131,6 +131,16 @@ def remove_small_segments(segments: np.ndarray, min_area: int = 3) -> np.ndarray
     return valid_segments
 
 
+def format_raster_data(raster, padding_mask=None, nodata=np.iinfo(np.uint8).max):
```
Comment (jhkennedy) [inline asf_tools/water_map.py:134]: Might be worth adding a docstring here explaining what this function is doing

**Review by jhkennedy — CHANGES_REQUESTED** [review]
Looks pretty good! We should at least change the CHANGELOG

**Comment by jhkennedy** [comment]
To test: 
- [ ] make a watermap and confirm that the GeoTIFF is written as expected
- [ ] create a HAND vrt and check that in the XML, the tiff paths are correct (pointing to the new `glo-30-hand` bucket).

**Comment by jhkennedy** [comment]
Also needs #155

**Review by jhkennedy — APPROVED** [review]
Alright, everything looks good! :shipit: if the two tests I suggested :arrow_up: pass

**Comment by jacquelynsmale** [comment]
@jhkennedy  the hand_file_paths looks liek ['/vsicurl/https://glo-30-hand.s3-us-west-2.amazonaws.com/v1/2021/Copernicus_DSM_COG_10_N38_00_W089_00_HAND.tif', '/vsicurl/https://glo-30-hand.s3-us-west-2.amazonaws.com/v1/2021/Copernicus_DSM_COG_10_N38_00_W090_00_HAND.tif', '/vsicurl/https://glo-30-hand.s3-us-west-2.amazonaws.com/v1/2021/Copernicus_DSM_COG_10_N38_00_W091_00_HAND.tif', '/vsicurl/https://glo-30-hand.s3-us-west-2.amazonaws.com/v1/2021/Copernicus_DSM_COG_10_N38_00_W092_00_HAND.tif', '/vsicurl/https://glo-30-hand.s3-us-west-2.amazonaws.com/v1/2021/Copernicus_DSM_COG_10_N39_00_W089_00_HAND.tif', '/vsicurl/https://glo-30-hand.s3-us-west-2.amazonaws.com/v1/2021/Copernicus_DSM_COG_10_N39_00_W090_00_HAND.tif', '/vsicurl/https://glo-30-hand.s3-us-west-2.amazonaws.com/v1/2021/Copernicus_DSM_COG_10_N39_00_W091_00_HAND.tif', '/vsicurl/https://glo-30-hand.s3-us-west-2.amazonaws.com/v1/2021/Copernicus_DSM_COG_10_N39_00_W092_00_HAND.tif', '/vsicurl/https://glo-30-hand.s3-us-west-2.amazonaws.com/v1/2021/Copernicus_DSM_COG_10_N40_00_W089_00_HAND.tif', '/vsicurl/https://glo-30-hand.s3-us-west-2.amazonaws.com/v1/2021/Copernicus_DSM_COG_10_N40_00_W090_00_HAND.tif', '/vsicurl/https://glo-30-hand.s3-us-west-2.amazonaws.com/v1/2021/Copernicus_DSM_COG_10_N40_00_W091_00_HAND.tif', '/vsicurl/https://glo-30-hand.s3-us-west-2.amazonaws.com/v1/2021/Copernicus_DSM_COG_10_N40_00_W092_00_HAND.tif']
, 
so I'm happy with that!

**Pull request merged**